ddtrace/contrib/grpc/server_interceptor.py

```diff
@@ -0,0 +1,32 @@
+import grpc
+
+from ddtrace import Pin
+from ...propagation.http import HTTPPropagator
+
+class GrpcServerInterceptor(grpc.ServerInterceptor):
+
+    def __init__(self):
+        self._pin = Pin.get_from(grpc)
+        self._propagator = HTTPPropagator()
+
+    def intercept_service(self, continuation, handler_call_details):
+        if not self._pin or not self._pin.enabled():
+            return continuation(handler_call_details)
+
+        self.extract_and_set_context(handler_call_details.invocation_metadata)
+
+        with self.start_span(handler_call_details.method) as span:
+            try:
```

Comment:
This `try/except` shouldn't be necessary, the span context manager will pick up on any exceptions and set the traceback appropriately